docs/user-guide.md

```diff
@@ -118,7 +118,7 @@ strategies:
 
 #### Balance low utilization nodes
 Using `HighNodeUtilization`, descheduler will rebalance the cluster based on memory by evicting pods
-from nodes with memory utilization lower than 20%. This should be used along with scheduler strategy `MostRequestedPriority`.
+from nodes with memory utilization lower than 20%.
```

Comment:
same as above, we don't just want to remove MostRequestedPriority; need to update with the new change